.github/workflows/enos-run.yml

```diff
@@ -283,9 +276,18 @@ jobs:
           ENOS_VAR_test_email: ${{ secrets.SERVICE_USER_EMAIL }}
           ENOS_VAR_boundary_docker_image_name: ${{ inputs.docker-image-name }}
           ENOS_VAR_boundary_docker_image_file: ./support/boundary_docker_image.tar
+          ENOS_DEBUG_DATA_ROOT_DIR: ./support/debug-data
         run: |
           export ENOS_VAR_enos_user=$GITHUB_ACTOR && \
           enos scenario run --timeout 60m0s --chdir ./enos ${{ matrix.filter }}
+      - name: Upload Debug Data
+        if: failure()
+        uses: actions/upload-artifact@0b7f8abb1508181956e8e162db84b466c27e18ce # v3.1.2
+        with:
+          # The name of the artifact is the same as the matrix scenario name with the spaces replaced with underscores and colons replaced by equals.
+          name: ${{ steps.prepare_scenario.outputs.debug_data_artifact_name }}
+          path: ./enos/support/debug-data
```

Comment:
we can use `${{ env. ENOS_DEBUG_DATA_ROOT_DIR }}` ?

Reply:
That env var is only defined in the steps that actually run enos.

Reply:
since our variable is constant we can just define it at job level and use it.

Reply:
> since our variable is constant we can just define it at job level and use it.

good call... I'll add it as a job level env var and then use your suggestion.